CMSgov/design-system
# [WNMGDS-2941] Remove clickable card guidance from button component (mgov)

## Summary

- Removes clickable card guidance from Button component on medicare theme, per https://jira.cms.gov/browse/WNMGDS-2941

## How to test

1. Look at medicare theme and ensure that "Note that clickable cards are sentence case. See that component for more information." is no longer visible on the Button component. It should also not exist on any other theme.

## Checklist
